Add `Generate` button in the navbar

Add a `Generate` button in the navbar.
Button gradient should be same as **logo**.

> > > Heyy @Gaurav-Verma07 , do i have to implement any functionality to the button?
> > 
> > 
> > kasa run karaych program? maza hot nahiye, mala pn conrobute karaych ahe
> 
> Please reply in english

i also want to contribute to other issues, but im having trouble configuring project, during npm install lots of errors are comming, i have never worked on TS, so i dont know how to load project, i tried many solutions from internet, but none work...please help
<pre>
PS E:\ramdon work\syntData\synd_dev_CODEPEAK> npm install
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: @typescript-eslint/eslint-plugin@6.12.0
npm WARN Found: @typescript-eslint/parser@5.59.7
npm WARN node_modules/@typescript-eslint/parser
npm WARN   dev @typescript-eslint/parser@"^5.57.1" from the root project
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer @typescript-eslint/parser@"^6.0.0 || ^6.0.0-alpha" from @typescript-eslint/eslint-plugin@6.12.0
npm WARN node_modules/@typescript-eslint/eslint-plugin
npm WARN   dev @typescript-eslint/eslint-plugin@"^6.12.0" from the root project
npm WARN   1 more (eslint-config-standard-with-typescript)
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: @typescript-eslint/eslint-plugin@6.12.0
npm WARN Found: @typescript-eslint/parser@5.59.7
npm WARN node_modules/@typescript-eslint/parser
npm WARN   dev @typescript-eslint/parser@"^5.57.1" from the root project
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer @typescript-eslint/parser@"^6.0.0 || ^6.0.0-alpha" from @typescript-eslint/eslint-plugin@6.12.0
npm WARN node_modules/@typescript-eslint/eslint-plugin
npm WARN   dev @typescript-eslint/eslint-plugin@"^6.12.0" from the root project
npm WARN   1 more (eslint-config-standard-with-typescript)
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: @typescript-eslint/eslint-plugin@6.12.0
npm WARN Found: @typescript-eslint/parser@undefined
npm WARN node_modules/@typescript-eslint/parser
npm WARN   dev @typescript-eslint/parser@"^5.57.1" from the root project
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer @typescript-eslint/parser@"^6.0.0 || ^6.0.0-alpha" from @typescript-eslint/eslint-plugin@6.12.0
npm WARN node_modules/@typescript-eslint/eslint-plugin
npm WARN   dev @typescript-eslint/eslint-plugin@"^6.12.0" from the root project
npm WARN   1 more (eslint-config-standard-with-typescript)
npm ERR! code ENOTFOUND
npm ERR! syscall getaddrinfo
npm ERR! errno ENOTFOUND
npm ERR! network request to https://registry.npmjs.org/@typescript-eslint%2fparser failed, reason: getaddrinfo ENOTFOUND your-proxy-server
npm ERR! network This is a problem related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Sami\AppData\Local\npm-cache\_logs\2023-12-11T08_59_55_790Z-debug-0.log 
</pre>

--- a/src/components/Header/Header.tsx
+++ b/src/components/Header/Header.tsx
@@ -124,6 +124,14 @@ export function HeaderMenu() {
               size="sm"
               hiddenFrom="sm"
             />
+            <Title
+              className={`gradient  ${classes.logo} `}
+              onClick={() => {
+                navigate("/tool");
+              }}
+            >
+              Generate
+            </Title>
           </div>
         </Container>
       </header>

